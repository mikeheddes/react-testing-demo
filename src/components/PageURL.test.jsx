// import React from "react";
// import { shallow } from "enzyme";
// import renderer from "react-test-renderer";
// import 'jest-styled-components';
