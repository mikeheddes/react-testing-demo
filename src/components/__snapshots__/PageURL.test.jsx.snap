// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageURL wrapped snapshot 1`] = `
.c1 {
  background-color: #FAA138;
  position: relative;
  box-shadow: 8px 8px rgba(0,0,0,0.15);
}

.c2 {
  display: block;
  text-align: center;
  color: white;
  font-weight: 500;
  font-size: 42px;
  padding: 39px 50px;
}

.c0 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  alt="Click for new random url"
  className="c0"
  href="/c0kio"
  onClick={[Function]}
  title="Click for new random url"
>
  <div
    className="c1"
    color="#FAA138"
  >
    <span
      className="c2"
    >
      URL: 
      /en
    </span>
  </div>
</a>
`;
